### DISCUSSION TITLE
[A-L] (2023/24) Foglio 9 - Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#324] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-17 185125](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/dabb9635-2031-4de3-a930-4f90a9cfa1b0)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/b971d305-7018-4bcc-ab63-cb94fe881983)
** Esercizio "ampliato"
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/38002461-9a86-4703-bb88-80461a1f0437)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale

![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/6a370f0c-db5b-425e-a8ef-435ac85f8e3e)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/2d288ff6-c21c-490b-8587-c6fd2e7ca249)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/f1c6e43d-94d0-418b-a13c-4391e068a427)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/4c3cf950-ee17-4927-8fdd-54e626f03aeb)
